extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -91,32 +91,40 @@ function render_player( $track_list, $attributes ) {
 	}
```

Comment:
Now that we rely on `get_track_list` from the API endpoint as well, I wonder if it may be worth mentioning that in the docblock before the function, so folks know the impact that changing that function can have?